src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedDayNames.cs

```diff
@@ -10,70 +10,30 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoAbbreviatedDayNames
     {
-        // PosTest1: Call AbbreviatedDayNames getter method should return correct value for InvariantInfo
         [Fact]
-        public void TestGetter()
+        public void AbbreviatedDayNames_InvariantInfo()
         {
-            VerificationHelper(DateTimeFormatInfo.InvariantInfo, new string[] {
-                    "Sun","Mon","Tue","Wed","Thu","Fri","Sat"}, false);
+            Assert.Equal(new string[] { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" }, DateTimeFormatInfo.InvariantInfo.AbbreviatedDayNames);
         }
 
-        // PosTest2: Call AbbreviatedDayNames setter method should return correct value
         [Fact]
-        public void TestSetter()
+        public void AbbreviatedDayNames_Set()
         {
-            VerificationHelper(new DateTimeFormatInfo(), new string[] {
-                    "1","2","3","4","5","6","7"}, true);
+            string[] newAbbreviatedDayNames = new string[] { "1", "2", "3", "4", "5", "6", "7" };
+            var format = new DateTimeFormatInfo();
+            format.AbbreviatedDayNames = newAbbreviatedDayNames;
+            Assert.Equal(newAbbreviatedDayNames, format.AbbreviatedDayNames);
         }
 
-        // NegTest1: ArgumentNullException should be thrown when The property is being set to a null reference
         [Fact]
-        public void TestNullProperty()
+        public void AbbreviatedDayNames_Set_Invalid()
         {
-            Assert.Throws<ArgumentNullException>(() =>
-            {
-                new DateTimeFormatInfo().AbbreviatedDayNames = null;
-            });
+            Assert.Throws<ArgumentNullException>(() => new DateTimeFormatInfo().AbbreviatedDayNames = null); // Value is null
+            Assert.Throws<ArgumentNullException>(() => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
+            Assert.Throws<ArgumentException>(() => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "sun" }); // Value.Length is not 13
```

Comment:
I assume the problem is that value.Length is not 7; as opposed to the stated 13?